examples/natural_language_understanding_v1.py

```diff
@@ -0,0 +1,30 @@
+
+# coding: utf-8
+
+# In[4]:
+
+import sys
+import os
+sys.path.append(os.path.join(os.getcwd(),'..'))
+import watson_developer_cloud
+import watson_developer_cloud.nlu.features.v1 as features
+
+
+# In[3]:
+
+nlu = watson_developer_cloud.NaturalLanguageUnderstandingV1(version='2016-01-23',
+                                                            username='USERNAME',
+                                                            url='https://gateway-s.watsonplatform.net/natural-language-understanding/api',
```

Comment:
In the public example we probably shouldn't use gateway-s, even though the public gateway doesn't work yet.